Text loading

cammac60 opened the pull request:
## This commit:

- Adds the text loading component and replaces the Loader with it when loading characters.

## Where to look: 

- `App.js` lines 130-150.
- The entire `Intro` directory houses the Intro component/files.

JamesRexMiller4 commented:
Looking at the images folder of this branch the image you are sourcing to does not appear to be in there at all. : (

cammac60 commented:
@JamesRexMiller4 Sorry about that, I thought for sure I added it but I must've forgot. I just added another commit and the file should be included now.

JamesRexMiller4 commented:
thanks! branch is working now. PR approved
